WordPress/gutenberg
# fixed issue of Font Size animates on appearance change

## What?

In site editor sidebar, width of Font size button within the Typography change while appearance drop-down is open. I have looked into details. and found that scrollbar is causing this issue. when we open the dropdown it goes outside of screen vertically and get scrollbar into the picture. and with scrollbar, container div size get reduced by 16px. 

Fixes https://github.com/WordPress/gutenberg/issues/40708.
 
## Testing Instructions





1.  Go to Site Editor  
2.  Go to Styles.
3.  Typography -> Text.
4.  Change the Appearance from the dropdown.
5.  Check that font size button width is same as before.
